src/main/webapp/feedback/load-icon-feedback.js

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TimestampUtil from '../timestamp-util.js';
+import IconFeedbackUtil from './icon-feedback-util.js';
+import ParsedIconFeedback from './parsed-icon-feedback.js';
+
+/* Handles loading IconFeedback from database and parses data for graph. */
+export default class LoadIconFeedback {
+  static #ENDPOINT_FEEDBACK = '/icon-feedback';
+  static #PARAM_LECTURE_ID = 'lectureId';
+
+  /* Each interval is 10 seconds, used to increment interval. */
+  static #INCREMENT_INTERVAL_MS = 10000;
+
+  #lectureId;
+  #parsedIconFeedback;
+
+  constructor(lectureId) {
+    this.#lectureId = lectureId;
+  }
+
+  async initialize() {
+    this.#parsedIconFeedback = new ParsedIconFeedback();
+    await this.loadIconFeedbackList();
+  }
+
+  /**
+   * Fetches available IconFeedback from `ENDPOINT_FEEDBACK`
+   * and parses the data for it to be graphed.
+   */
+  async loadIconFeedbackList() {
+    const url =
+        new URL(LoadIconFeedback.#ENDPOINT_FEEDBACK, window.location.origin);
+    url.searchParams.append(
+        LoadIconFeedback.#PARAM_LECTURE_ID, this.#lectureId);
+    const response = await fetch(url);
+    const jsonData = await response.json();
+    console.log(jsonData);
+    this.parseFeedback(jsonData);
+  }
+
+  /**
+   * Parses `iconFeedbackJson` by couting how many times each IconFeedback type
+   * is clicked in each 10 second interval and stores that data in a
+   * ParseIconFeedback object.
```

Comment:
This talks about internal code. A JSDoc, JavaDoc should be high level so that if someone who does not have visibility into the code understands what this does. Imagine if you were reading the JavaDoc for HashMaps(https://docs.oracle.com/javase/8/docs/api/java/util/HashMap.html) and it talked about internal code. That would be of no use to you.